builder/main.py

```
@@ -297,6 +561,419 @@ def check_lib_archive_exists():
+            print(f"  Successfully detected SPIFFS configuration {i}")
+            
+            return {
+                'page_size': config['page_size'],
+                'block_size': config['block_size'],
+                'obj_name_len': config['obj_name_len'],
+                'meta_len': 4,
+                'use_magic': True,
+                'use_magic_len': True,
+                'aligned_obj_ix_tables': False
+            }
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)